comparisontool/urls.py

```diff
@@ -12,13 +12,6 @@
 )
 
 urlpatterns = [
-    re_path(
-        r'^$',
-        TemplateView.as_view(
-            template_name='comparisontool/landing.html'
```

Comment:
Can [landing.html](https://github.com/cfpb/django-college-costs-comparison/blob/main/comparisontool/templates/comparisontool/landing.html) itself also be deleted?

Reply:
yes, I have some more work to do to delete tests and the items you mention.